intx_sdk/acquire_repay_loan.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2025-present Coinbase Global, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from dataclasses import dataclass, asdict
+from typing import List, Optional
+from intx_sdk.base_response import BaseResponse
+from intx_sdk.client import Client
+from intx_sdk.credentials import Credentials
+
+
+@dataclass
+class AcquireRepayLoanRequest:
```

Comment:
in ts, I made this acquireOrRepayLoan . the slash doesn't translate to object names, and that was a better middle ground